reference/unity_to_godot.rst

```diff
@@ -0,0 +1,187 @@
++===================+===================================================================================+================================================================================================================+
+| License           | Proprietary, closed, free license with revenue caps and usage restrictions        | MIT License,  free and fully open souce without any restriction                                                |
++-------------------+-----------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------+
+| OS (editor)       | Windows, OSX, Linux (unofficial and unsupported)                                  | Windows, X11 (Linux, *BSD), Haiku, OSX                                                                         |
++-------------------+-----------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------+
+| OS (export)       | | Desktop: Windows, Linux/SteamOS, OSX                                            | | Desktop: Windows, X11, OSX                                                                                   |
+|                   | | Mobile: Android, iOS, Windows Phone, Tizen,                                     | | Mobile: Android, iOS, Blackberry (deprecated)                                                                |
+|                   | | Web: WebGL                                                                      | | Web: WebGL, HTML5 (via emscripten, broken)                                                                   |
+|                   | | Consoles: PS4, PS Vita, XBox One, XBox 360, WiiU, 3DS                           |                                                                                                                |
+|                   | | VR: Occulus Rift, SteamVR, Google Cardboard, Playstation VR, Gear VR, HoloLens  |                                                                                                                |
+|                   | | TV: AndroidTV, Samsung SMARTTV, tvOS                                            |                                                                                                                |
++-------------------+-----------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------+
+| Scene system      | | Entity-Component System (ECS) (GameObject > Component).                         | Scene tree and nodes, allowing scenes to be nested and/or inherit other scenes                                 |
```

Comment:
Just as a note, Unity is not ECS in any way shape or form, it is entirely Component/Scene based.  ECS is where you have data strongly separated from code, most generally in a Dataflow pattern (since ECS is a superset of Dataflow after all).  Unity has absolutely no Dataflow style in it whatsoever (which is one of its major speed hits).  Unity's design is entirely ComponentScene.